src/data/intel_icelake_perf_events.rs

```diff
@@ -0,0 +1,40 @@
+use crate::data::perf_stat::{NamedCtr, NamedTypeCtr, PerfType};
+
+static CYCLES: NamedTypeCtr = NamedTypeCtr {
```

Comment:
These are same as the default in intel_perf_events.rs. So, does this architecture specific config file provide delta counters or all?

Reply:
We make sure the nr and dr are in the same file. We don't want to refer to other files for a PMU event.